src/tonuino.cpp

```diff
@@ -191,9 +193,28 @@ void Tonuino::disableStandbyTimer() {
   }
 }
 
+void Tonuino::enableAmplifier() {
+#if defined ALLinONE || defined ALLinONE_Plus
+  // if (!amplifierEnabled){
+    digitalWrite(ampEnablePin, getLevel(ampEnablePinType, level::active));
+    amplifierEnabled = true;
```

Comment:
You should remove this line and the variable at all. It increases only flash and RAM usage.

Reply:
Okay, I wasn't so sure about the effects of writing the same value to the pin twice.